.github/workflows/ansible-tests.yml

```diff
@@ -21,15 +21,9 @@ jobs:
     strategy:
       # https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix
       matrix:
-        ansible: ["2.14", "2.15", "2.16"]
-        python: ["3.8", "3.9", "3.10", "3.11"]
+        ansible: ["2.15", "2.16"]
+        python: ["3.9", "3.10", "3.11"]
```

Comment:
Can we add `3.12` also? 

According to the [release matrix](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix) `2.16` should be able to use it.

Then you should also exclude
```
- ansible: "2.15"
  python: "3.12"
```